R-packages/evalcast/R/evalcast-package.R

```diff
@@ -22,9 +22,11 @@ NULL
 .onLoad <- function(libname, pkgname) {
     download_signal <<- memoise::memoise(download_signal)
     get_covidhub_predictions <<- memoise::memoise(get_covidhub_predictions)
+    get_forecaster_predictions <<- memoise::memoise(get_forecaster_predictions)
     msg <- c(
-        "Calls to the `evalcast` functions `download_signal()` and",
-        "`get_covidhub_predictions()` are memoized in memory by default.",
+        "Calls to the `evalcast` functions `download_signal()`,",
+        "`get_covidhub_predictions()`, and `get_forecast_predictions` are",
```

Comment:
Add trailing parens to the function name

Reply:
Fixed.